solr/CHANGES.txt

```diff
@@ -1578,7 +1583,7 @@ Improvements
   SOLR_REQUESTLOG_ENABLED=true. This will respect SOLR_LOGS_DIR if it is set. (rmuir)
 
 * SOLR-14136: Jetty's InetAccessHandler can be enabled via environment variables,
-  SOLR_IP_WHITELIST and SOLR_IP_BLACKLIST. These variables can restrict access to
+  SOLR_IP_ALLOWLIST and SOLR_IP_DENYLIST. These variables can restrict access to
```

Comment:
Does this need to be changed?

Reply:
Nope, this is a search/replace error. Will revert.

Reply:
Done